devfiles/java-web-vertx/devfile.yaml

```diff
@@ -0,0 +1,65 @@
+---
+apiVersion: 1.0.0
+metadata:
+  name: java-web-vertx
+projects:
+  -
+    name: java-web-vertx
+    source:
+      type: git
+      location: "https://github.com/che-samples/web-java-vertx"
+components:
+  -
+    type: chePlugin
+    id: redhat/java/latest
+  -
+    type: dockerimage
+    alias: maven
+    image: registry.centos.org/che-stacks/centos-jdk8
+    command: ['sleep']
+    args: ['infinity']
+    env:
```

Comment:
This environment looks similar to how the old version of the other examples looked like. IMHO, it could be simplified to something like:

```yaml
    env:
      - name: JAVA_OPTS
        value: "-XX:MaxRAMPercentage=50.0 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10
          -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90
          -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom
          -Duser.home=/home/user"
      - name: MAVEN_OPTS
        value: $(JAVA_OPTS)
```

Reply:
yep, fixed